src/components/full-calender/actions/index.js

```diff
@@ -43,10 +43,9 @@ export const priorityUpdateCheckbox = payload=>dispatch=>{
 	
 };
 
-export const fetchProrityEvents = payload=>{
+export const fetchProrityEvents = ()=>{
 	return async dispatch=>{
-		const response = await getProrityData();
-		const payload = await response.json();
+		const payload = await getProrityData();
```

Comment:
if return value is not required to hold in a const ,update to
await getProrityData()